### ISSUE TITLE
AVT2 - Toggle Button Component does not activate when Enter is pressed 

### ISSUE THREAD

**Issue by snidersd** [IBM/carbon-components-react#2195]
## Environment
Tested on macOS Mojave 10.14.2, Chrome Version 71.0.3578.98 (Official Build) (64-bit), with VoiceOver

## Description
Without using a mouse navigate to the toggle button and press the Enter key to activate the button. It fails to respond. Also note, this issue occurs with and without a screen reader.

Expected result: This should not be a checkbox. It should be implemented as a toggle button based on [ARIA Authoring Practices 1.1](https://www.w3.org/TR/wai-aria-practices-1.1/#button)  the Enter or Space key should toggle the button, but only the Space key works. Note: This issue was also discussed with members of the [W3C WAI-ARIA working group](https://github.com/w3c/aria-practices/issues/1016).

**Comment by paschalidi** [comment]
https://github.com/IBM/carbon-components-react/pull/2219 this is in progress
